ValetWatchTest Extension/ExtensionDelegate.h

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
This class (and it's .m) also look like boilerplate. Do we need them? Are you planning on adding functionality here?